app-misc/neofetch/neofetch-3.3.0.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="A CLI system information tool written in BASH that supports displaying images"
+DESCRIPTION="Neofetch displays information about your system next to an image, your OS logo, or any ASCII file of your choice. The main purpose of Neofetch is to be used in screenshots to show other users what OS/Distro you're running, what Theme/Icons you're using etc"
+HOMEPAGE="https://github.com/dylanaraps/neofetch"
+COPYRIGHT="2016-2017 Dylan Araps"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/dylanaraps/$portName/archive/$portVersion/$portVersionedName.tar.gz"
+CHECKSUM_SHA256="4808e76bd81da3602cb5be7e01dfed8223b1109e2792755dd0d54126014ee696"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	neofetch
+	cmd:neofetch
+	"
+REQUIRES="
+	haiku
+	cmd:bash
+	"
+BUILD_PREQUIRES="
```

Comment:
We like to have a blank line before BUILD_PREQUIRES.
(All the rest is OK)
Sorry, I should have given the link to the wiki with my first comment about cosmetic changes ;-)
[HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)